.circleci/config.yml

```diff
@@ -363,7 +363,7 @@ jobs:
       - <<: *nats-streaming
       - <<: *memcached
     <<: *shared
-    parallelism: 16
+    parallelism: 20
```

Comment:
We have the option to either run them with low parallel config or high.
My thinking was: if I want to rebuild the job before a release, I want a fast result.
In the night, I don't care. TBD

Reply:
In that case, wouldn't it make sense to set a high parallelism for Node 16 ad 18, and a low one (maybe `parallelism: 1`) for the older Node versions that are only run nightly?

Reply:
> and a low one (maybe parallelism: 1) for the older Node versions that are only run nightly?

Yeah totally, but what about this:

> if I want to rebuild the job before a release (e.g. because it failed nightly)